ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +650,110 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_ocs(self):
+        """
+        The function Uninstalls the OCS operator from a openshift
+        cluster and removes all its settings and dependencies
+
+
+        Returns
+            Boolean: True if operation was successful
+
+        """
+        ocp_obj = ocp.OCP()
+        provisioners = constants.OCS_PROVISIONERS
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass(all_ns=True)
+        for storage_class in sc_list:
+            if storage_class.get('provisioner') in provisioners:
+                sc_list.remove(storage_class)
+
+        # STEP 2: Query for PVCs and OBCs that are using the storage class provisioners listed in the previous step.
+        pvc_list = get_all_pvcs(namespace='all-namespaces')
+        pvcs_to_delete = []
+        for pvc in pvc_list:
+            for storage_class in sc_list:
+                if pvc.backed_sc == storage_class:
+                    pvcs_to_delete.append(pvc)
+                    break
+
+        # Removing monitoring stack from OpenShift Container Storage (edit config map with patch)
+        monitoring_obj = ocp.OCP(namespace=constants.MONITORING_NAMESPACE,
+                                 kind='configmap',
+                                 resource_name='cluster-monitoring-config')
+        monitoring_list = monitoring_obj.get('data').get('config.yaml')
+        for item in monitoring_list:
+            if item.get().get('volumeClaimTemplate').get('spec').get('storageClassName') in sc_list:
+                monitoring_list.remove(item)
+        monitoring_obj.patch(resource_name='cluster-monitoring-config',
+                             params=f'{{"data": {{"config.yaml": {monitoring_list}}}}}')
+
+        # Removing OpenShift Container Platform registry from OpenShift Container Storage
+        image_registry_obj = ocp.OCP(namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE)
+        if self.platform == constants.AWS_PLATFORM:
+            image_registry_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                                     params=" '{\"spec\":{\"storage\":{}}}' ")
+
+        elif self.platform == constants.VSPHERE_PLATFORM:
+            image_registry_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                                     params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' ")
+
+        # Removing the cluster logging operator from OpenShift Container Storage
+        # Remove the ClusterLogging instance in the namespace.
+        ocp_obj.exec_oc_cmd("delete clusterlogging instance -n openshift-logging --wait=true --timeout=5m")
+
+        # STEP3: Follow these instructions to ensure that the PVCs listed in the previous step are deleted
+        # STEP 3.1: Determine the pod that is consuming the PVC.
+        pod_list = get_all_pods()
+        for pod in pod_list:
```

Comment:
Please create a function under pvc.py to get the pod/s attached to PVC and use it
something like:
```
    def get_attached_pods(self):
        """
        Get the pods attached to this PVC

        Returns:
             list: A list of pod objects attached to the PVC
        """
        attached_pods = []
        all_pods = POD.get_all_pods()
        for pod in all_pods:
            pvc = POD.get_pvc_name(pod)
            if pvc == self.name:
                attached_pods.append(pod)
        return attached_pods
```